ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -147,11 +161,14 @@ type opennebula_datastore = {
 
 type opennebula_vnet = {
     "name" : string
-    "type" : string  = 'FIXED'
```

Comment:
@jrha ok, I have included a new comment into PR description to clarify this point